spec/test_app/app/controllers/basechurch/v1/posts_controller_spec.rb

```diff
@@ -1,33 +1,40 @@
 require 'rails_helper'
 
 describe Basechurch::V1::PostsController, type: :controller do
-  let(:group) do
-    create(:group)
-  end
+  let(:group) { create(:group) }
 
-  let!(:users) { create_list(:user, 3) }
   let(:user) { create(:user) }
 
   let(:all_attributes) do
     {
-      posts: {
-        content: Forgery(:lorem_ipsum).words(10),
-        publishedAt: DateTime.now.to_time.iso8601,
-        tags: ['tag1', 'tag2', 'tag3'],
-        title: Forgery(:lorem_ipsum).title,
-        links: {
-          group: group.id.to_s
+      data: {
+        type: "posts",
+        attributes: {
+          content: Forgery(:lorem_ipsum).words(10),
+          publishedAt: DateTime.now.to_time.iso8601,
+          tags: ['tag1', 'tag2', 'tag3'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.